iocore/net/UnixNetAccept.cc

```diff
@@ -45,10 +43,14 @@ net_accept(NetAccept *na, void *ep, bool blockable)
   Event *e               = static_cast<Event *>(ep);
   int res                = 0;
   int count              = 0;
-  const int loop         = NetAccept::accept_till_done;
   UnixNetVConnection *vc = nullptr;
   Connection con;
 
+  EThread *t             = e->ethread;
+  NetHandler *handler    = get_NetHandler(t);
+  int config_value       = handler->config.additional_accepts;
```

Comment:
Could this value be put in the `NetAccept` because it's a process static, correct?

Reply:
not quite - it's a reloadable config, but I can still put the ternary operation in the `NetHandler` similar to where the config variable itself lives